packages/synapse-react-client/test/containers/OrientationBanner.test.tsx

```diff
@@ -0,0 +1,155 @@
+    const { alert, buttons } = setUp()
+    expect(alert).toBeInTheDocument()
+    expect(alert).toHaveTextContent(defaultProps.title)
+    expect(alert).toHaveTextContent(defaultProps.text)
+    expect(buttons.close).toBeInTheDocument()
+    expect(buttons.primary).toBeInTheDocument()
+    expect(buttons.secondary).toBeInTheDocument()
+  })
+
+  it('does not display the alert when the notification has been dismissed', () => {
+    localStorage.setItem(defaultStorageId, 'true')
+    const { alert } = setUp(defaultProps)
+    expect(alert).not.toBeInTheDocument()
+  })
+
+  it('displays the alert even if a different notification was previously dismissed', () => {
+    const otherBannerName = 'Challenges' as OrientationBannerName
+    localStorage.setItem(getOrientationBannerKey(otherBannerName), 'true')
+    renderComponent(defaultProps)
+    expect(screen.queryByRole('alert')).toBeInTheDocument()
+    localStorage.removeItem(getOrientationBannerKey(otherBannerName))
+  })
+
+  it('hides the alert when the close button is clicked', async () => {
+    const { alert, user, buttons } = setUp()
+    expect(alert).toBeInTheDocument()
+    expect(buttons.close).toBeInTheDocument()
+    user.click(buttons.close!)
+    await waitForElementToBeRemoved(alert)
+    expect(alert).not.toBeInTheDocument()
+  })
+
+  it('does not have link or action buttons when neither are provided', () => {
+    const { buttons } = setUp({
+      ...defaultProps,
+      primaryButtonConfig: undefined,
+      secondaryButtonConfig: undefined,
+    })
+    expect(buttons.primary).not.toBeInTheDocument()
+    expect(buttons.secondary).not.toBeInTheDocument()
+  })
+
+  it('has a link button when only a link is provided and has the correct href attribute', async () => {
+    const { user, buttons } = setUp({
+      ...defaultProps,
+      primaryButtonConfig: undefined,
+    })
+    expect(buttons.primary).not.toBeInTheDocument()
+    expect(buttons.secondary).toBeInTheDocument()
+
+    await user.click(buttons.secondary as HTMLElement)
+    expect(window.open).toHaveBeenCalledTimes(1)
+    expect(window.open).toHaveBeenCalledWith(
+      // @ts-ignore - typescript doesn't recognize href within the conditional AlertButtonConfig type
+      defaultProps.secondaryButtonConfig!.href,
+      '_blank',
+      'noopener',
+    )
+  })
+
+  it('has an action button when only an acton is provided', () => {
+    const { buttons } = setUp({
+      ...defaultProps,
+      secondaryButtonConfig: undefined,
+    })
+    expect(buttons.primary).toBeInTheDocument()
+    expect(buttons.secondary).not.toBeInTheDocument()
+  })
+
+  it('calls the correct callback when the primary button is clicked', async () => {
+    const { user, buttons } = setUp()
+    expect(onPrimaryButtonClicked).toHaveBeenCalledTimes(0)
+    expect(buttons.primary).toBeInTheDocument()
+    await user.click(buttons.primary as HTMLElement)
+    expect(onPrimaryButtonClicked).toHaveBeenCalledTimes(1)
+  })
```

Comment:
These tests aren't really necessary if these cases are already tested in FullWidthAlert, since you are really just passing them along.

Reply:
True! We didn't have an existing set of tests for FullWidthAlert, so I'll create some and move these tests into that file.